menu/freedesktop/asebachallenge.desktop

```diff
@@ -9,5 +9,5 @@ GenericName[es]=Aseba simulador de robot para jugar en la red
 GenericName[pt]=Aseba simulador de robô para jogar em rede
 Exec=asebachallenge
 TryExec=asebachallenge
-Categories=Education;Science;Aseba
+Categories=Education;Science;Aseba;
```

Comment:
If I am using something non-standard, let's correct it! The reason I used the Aseba category was to avoid polluting the higher-level one with infrequently-used programs. But it is fine to remove it.